text/0000-temp-config-settings.md

```diff
@@ -13,65 +13,368 @@ nit-picking the "gotchas" in code examples. :heart:
+> recommended that teams use our React library for the "end all solution".
+
+### jsxFactory = `pando`
+
+This will create the jsx components to use the naming convention of:
+
+```jsx
+<pando.div />
+```
 
 # Drawbacks
 
-Why should we *not* do this? Please consider:
+With any change, there are tradeoffs. Especially when trying to create structure
+where there is none.
+
+## Why should we _not_ do this?
 
 - implementation cost, both in term of code size and complexity
 - whether the proposed feature can be implemented in user space
 - the impact on teaching people a Pando library
 - integration of this feature with other existing and planned features
 - cost of migrating existing Pando applications (is it a breaking change?)
 
-There are tradeoffs to choosing any path. Attempt to identify them here.
+### Implementation cost
+
+This is dependant on the team. Right now, we are seeing that each product team
+manages their project and tech stack in it's own way which leads to a lot of duplicate
+code and unnecessary complexity.
+
+What is meant by this is that Skills, Flow, and Social and Hands-on Learning all have their own
+way of managing their styles including teams within product teams. This means that
+there is no way of really measuring the implementation cost on the Pando side
+because there is currently no standadard.
+
+For the sake of the argument, let's say that the implementation cost is high.
+
+> :warning: **Note:** This is ultimately the goal to resolve via Pando. The more
+> we can standardize the way we do things, the more we can reduce the implementation
+> cost and the more we can focus on the product.
+
+### User space implementation
+
+Right now, every team uses React across the company (outside of one or two Skills
+BC's) so this is not a concern. Likewise, most teams are using CSS-in-JS so this
+enhancement should be easy to pick up and migrate. For the most part, teams should
+be reducing code by adopting and using Pando React.
+
+Panda :panda: is becoming _very_ popular across the across the industry for this exact
+reason. It is easy to pick up and use.
+
+### Impact on teaching
+
+Documentation shouldn't be too affected by this change. The only thing that would
+change is the way we teach the installation process and how to use Pando React
+via the new customizations it will offer.
+
+### Integration with other features
+
+No different here than just getting teams to use the Pando React library over
+the current chaos. Once again, this is a good thing because it will allow us to
+create a more consistent experience (both customer and developer) across the company.
+
+### Cost of migration
+
+This would introduce a breaking change **only** for teams that have yet to adopt
+Pando React. For teams that have already adopted Pando React, this would be a
+non-breaking change since the tooling is internal and it's just adding some deps
+and a config file.
```

Comment:
While this may be true, I think it should be noted that since originally there wasn't planned to be a component library, many teams currently utilizing pando may still not be on Pando React yet. It's probably worth it to explain more specifically _how_ this would break things for those not yet on the component lib, and not consider them essentially as outliers.

Also, what about non-react teams? Would it break things for the team(s) using svelte too?